Testing/SystemTests/tests/framework/WishAnalysis.py

```diff
@@ -64,9 +64,8 @@ def runTest(self):
         # align detectors
         ConvertUnits(InputWorkspace="w16748-1", OutputWorkspace="w16748-1", Target="TOF")
         ReplaceSpecialValues(InputWorkspace="w16748-1", OutputWorkspace="w16748-1", NaNValue="0", InfinityValue="0")
-        AlignDetectors(
-            InputWorkspace="w16748-1", OutputWorkspace="w16748-1", CalibrationFile="wish_grouping_noends2_no_offsets_nov2009.cal"
-        )
+        ApplyDiffCal(InstrumentWorkspace="w16748-1", CalibrationFile="wish_grouping_noends2_no_offsets_nov2009.cal")
+        ConvertUnits(InputWorkspace="w16748-1", OutputWorkspace="w16748-1", Target="dSpacing")
```

Comment:
Think this test reference workspace is actually converted back to TOF so need to run `ApplyDiffCal` with `ClearCalibration=True` after L68 (or anywhere before the call to `ConvertUnits` back to TOF